mobile/lib/modules/backup/services/backup.service.dart

```diff
@@ -196,6 +196,16 @@ class BackupService {
         : candidates.whereNot((e) => existing.contains(e.id)).toList();
   }
 
+  // AssetType::audio is greater than image / video
+  // But since we do not support them, it shouldn't be an issue
+  int _sortByAssetType(AssetEntity a, AssetEntity b) {
+    return a.typeInt < b.typeInt
```

Comment:
Simply `a.typeInt - b.typeInt` should work the same

Reply:
oh my, that's true. Let me update this real fast. thank you!